src/App.vue

```diff
@@ -1,55 +1,492 @@
+}
+.editor{
+  padding:12px 16px;
+  border-top: solid 1px gray;
+  border-bottom: solid 1px gray;
+}
+.editor-control{
+  display: flex;
+  justify-content: flex-end;
+}
+.twitter-search{
+  position:absolute;
+  color:#6E767D;
+  text-align:center;
+  border:solid;
+  border-color:#202327;
+  background-color: #202327;
+  border-radius: 80px;
+  margin-left:25px;
+  margin-top:10px;
+  padding-top:10px;
+  padding-bottom:10px;
+  width: 320px;
+}
+.trends-for-you{
+  color:#D9D9D9;
+  border:solid;
+  border-color:#202327;
+  border-radius: 25px;
+  margin-left:25px;
+  margin-top:70px;
+  padding-top:10px;
+  padding-bottom:10px;
+  padding-left: 20px;
+  padding-right: 20px;
+  width: 320px;
+  font-weight: bold;
+  background-color:#202327;
+}
+.who-to-follow{
+  color:#D9D9D9;
+  border:solid;
+  border-color:#202327;
+  border-radius: 25px;
+  margin-left:25px;
+  margin-top:20px;
+  padding-top:10px;
+  padding-bottom:10px;
+  padding-left: 20px;
+  padding-right: 20px;
+  width: 320px;
+  font-weight: bold;
+  background-color:#202327;
+}
+.tweet-container{
+  display:flex;
+  min-height:113px;
+  padding:12px 16px;
+  border-bottom:solid 1px gray;
+}
+.bird{
+  margin-right:12px;
+}
+.user-info{
+  display:flex;
+  font-size:15px;
+}
+.user-name{
+  margin-right:3.25px;
+  font-weight:700;
+}
+.authorize-mark{
+  margin-right:3.25px;
+}
+.tweet-date{
+  color:gray;
+}
+.action-bottun{
+  display:flex;
+  margin-top: 20px;
+  color: gray;
+}
+.reply{
+  margin-right: 20px;
+}
+.reply-number{
+  margin-right: 40px;
+}
+.favorite{
+  margin-right: 20px;
+}
+.favorite-number{
+  margin-right: 40px;
+}
+.retweet{
+  margin-right: 20px;
+}
+.retweet-number{
+  margin-right: 40px;
+}
+.trends-line1{
+  font-size: 13px;
+  color: gray;
+  display: flex;
+  justify-content: space-between;
+}
+.trends-line2{
+  font-size: 15px;
+}
+.trends-line3{
+  font-size: 13px;
+  color: gray;
+}
+.trends-1{
+  margin-top: 20px;
+}
+.trends-2{
+  margin-top: 20px;
+}
+.trends-3{
+  margin-top: 20px;
+}
+.trends-4{
+  margin-top: 20px;
+}
+.showmore{
+  margin-top: 20px;
+  font-weight: bold;
+  color: #1D9BF0;
+}
+.recommend-user{
+  display: flex;
+  margin-top: 20px;
+}
+.recommend-username{
   display: block;
-  margin: 0 auto 2rem;
+  color: white;
+  margin-left: 10px;
 }
-
-@media (min-width: 1024px) {
-  header {
-    display: flex;
-    place-items: center;
-    padding-right: calc(var(--section-gap) / 2);
-  }
-
-  .logo {
-    margin: 0 2rem 0 0;
-  }
-
-  header .wrapper {
-    display: flex;
-    place-items: flex-start;
-    flex-wrap: wrap;
-  }
+.recommend-usernameid{
+  color: #6E767D;
+  font-size: 15px;
+}
+.recommend-user-follow{
+  position: absolute;
+  margin-left: 10px;
+  margin-left: 235px;
+  border: solid 1px black;
+  border-radius: 25px;
+  padding: 3px 9px;
+  background-color: white;
+  color: black;
+  font-size: 13px;
+}
+.recommend-authorize-mark{
+  position: relative;
+  margin-bottom: 22px;
+}
+.myaccount{
```

Comment:
```suggestion
.my-account{
```
一貫性のため実際に英語でスペース入るなら`-`にしたほうがいい